### PR TITLE
Prevent pasting without `Copy or Move` on the target container

### LINKED ISSUE #4589: Probleme bei geschütztem Dossier mit Aufgabenzugriff
https://sentry.4teamwork.ch/sentry/onegov-gever/issues/8092/

Wird in einem geschützten Dossier, ein Benutzer mittels Aufgabe berechtigt, so wird ihm zwar die Möglichkeit geboten ein Dokument ins Dossier hinzuzufügen bzw. eine Dokument mittels  copy&paste einzufügen. Beim Versuch treten aber Berechtigungsproblem auf.

### PR DESCRIPTION
**Pull request by Rotonen**
* Amend the fixtures to have suitable data types
* Prevent pasting without `Copy or Move` on the target container as otherwise the freshly created object cannot be renamed

Closes #4589

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -605,6 +605,9 @@ Objects
             - self.meeting_subtask
         - self.protected_dossier
           - self.protected_document
+        - self.protected_dossier_with_task
```
Comment (phgross) [inline README.rst]: Why is an additional dossier with a task necessary? Whats the point to not just add the task to the protected_dossier

Comment (Rotonen) [reply README.rst]: So we have an otherwise protected dossier to which `self.regular_user` has access only because of the task assignation.

**Comment by Rotonen** [comment]
Juggling the rebases on this one are starting to eat unnecessary amounts of time in aggregate.

Additionally now the fixture test wants to have `self.mail` instead of `self.mail_eml` in there for me, and I cannot figure that one out.

```
diff --git a/README.rst b/README.rst
index 51d663caf..6c91611d4 100644
--- a/README.rst
+++ b/README.rst
@@ -577,7 +577,7 @@ Objects
           - self.draft_proposal
           - self.draft_word_proposal
           - self.empty_document
-          - self.mail
+          - self.mail_eml
           - self.mail_msg
           - self.proposal
             - self.proposaldocument
```

**Comment by jone** [comment]
@Rotonen what is the question?

**Comment by Rotonen** [comment]
@jone why does `bin/test -t TestDumpFixtureStrutureToReadme` fail on this branch?

**Comment by jone** [comment]
@Rotonen this is strange indeed.
Actually it is correct that there must be a `mail_eml`, it is actually registered here: https://github.com/4teamwork/opengever.core/blob/f1dc9ba3340b2c1c87d0522caf26d652fa28ef42/opengever/testing/fixtures.py#L1145

But I've no idea why this is not yet on master and why the master can be green without that 🤔

**Comment by Rotonen** [comment]
@jone the strangest thing there is it should want both in the readme, but it seems to want either or, based on a yet unknown pivot.

**Comment by jone** [comment]
I think the problem is here: https://github.com/4teamwork/opengever.core/blob/91d11052545de20383618405f1a3d76853399839/opengever/testing/tests/test_dump_fixture_structure.py#L51
A path-based dictionary of all fixture objects is built.
The problem is that `self.mail` and `self.mail_eml` is the same object, so there are two entries with the same path. Since dictionaries are involved, it is a bit random which one is the winner.

So we should either register an object only once (maybe add an assertion) or fix the dump test so that it supports multiple entries with the same path.

**Comment by Rotonen** [comment]
Ready for (re?)review.

**Comment by Rotonen** [comment]
Rebased and ready for (re?)review.

File: docs/HISTORY.txt [hunk 2]
```diff
@@ -32,6 +32,9 @@ Changelog
 - Prevent replacing files on proposal templates with non-.docx files via quickupload. [Rotonen]
 - Add an upgrade step to fix broken-by-broken-protocol-excerpt journal entries on dossiers. [Rotonen]
 - Do not list documents within dossiertemplates when creating a document from template. [njohner]
+- Fix task path queries for oracle backends. [phgross]
```
Comment (phgross) [inline docs/HISTORY.txt]: ✂️

File: docs/HISTORY.txt [hunk 3]
```diff
@@ -32,6 +32,9 @@ Changelog
...
+- Fix activity queries for oracle backends. [phgross]
```
Comment (phgross) [inline docs/HISTORY.txt]: ✂️

**Pull request merged**